src/charm.py

```diff
@@ -103,8 +103,14 @@ def _on_install(self, _) -> None:
         installs MongoDB.
         """
         self.unit.status = MaintenanceStatus("installing MongoDB")
-        self._add_mongodb_org_repository()
-        self._install_apt_packages(["mongodb-org"])
+        repo_name = "deb-https://repo.mongodb.org/apt/ubuntu-focal/mongodb-org/5.0"
+        repo_entry = "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse"
+        gpg_url = "https://www.mongodb.org/static/pgp/server-5.0.asc"
```

Comment:
Would echo @marcoppenheimer here about both points.

Also, since AFAIK we should support a number of os releases (focal, and others), these could be retrieved from the metadata.

Reply:
fully agree!

Reply:
As for updating version, I'll handle that in a separate PR in the future when it comes times to deal with updating and versioning.